frontend/src/pages/SessionForm/components/ParticipantsReadOnly.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
Do we have a test for this that I haven't gotten to yet?

Reply:
It's tested in participants.js. Most of this is just moved from that file as cleanup. I added an additional test to account for the new path of IST visit.

Reply:
nvm